README.md

```diff
@@ -122,16 +122,29 @@ For a quick and dirty setup, you can:
 1. Create a new user in the IAM service (Access Management > Users)
 2. Click "Attach policies directly" on the "Set permissions" page
 3. Add the "AWSLambda_FullAccess" and "IAMFullAccess" policies, and complete the user creation
-4. Create an access key for the user (don't worry about the warning)
-5. Place the access key and secret key in `~/.aws/credentials` (or wherever the
-   appropriate location is for your system):
+4. Create an access key for the user, with "Use case" "Application running on an AWS compute service" (don't worry about the warning - just confirm)
+5. Save your credentials in a password manager or in a `~/.aws/credentials` file (or wherever the
```

Comment:
One last note -- I specifically mention the `credentials` file because that's where `cargo-lambda` looks when deploying if those variables aren't available in the environment. Storing them in a password manager is a great idea, but might not make the intentions clear there. We could probably add a note to clarify this.